app.js

```diff
@@ -5,13 +5,16 @@ const {
   getEndpoints,
   getArticleById,
 } = require("./news.controllers");
-const { handleCustomErrors } = require("./error.controllers.js");
+const { handle400s, handleCustomErrors } = require("./error.controllers.js");
 
 app.get("/api/topics", getTopics);
+
 app.get("/api", getEndpoints);
+
 app.get("/api/articles/:article_id", getArticleById);
 
-app.use(handleCustomErrors);
```

Comment:
Why did you remove your custom errors here? We still need those for 404s in the database (as opposed to 404 paths in the url)